node-chrome/puppeteer-chrome-test.js

```diff
@@ -0,0 +1,65 @@
+const Apify = require('apify');
+const { version: puppeteerVersion } = require('puppeteer/package.json');
+const VERSION_REGEX = /([\d.?]+)/g;
+
+function parseCompatibilityVersions(text) {
```

Comment:
I'd just put a comment here with sample how it looks like

```
- Releases per Chromium Version:
  * Chromium 77.0.3803.0 - [Puppeteer v1.18.1](https://github.com/GoogleChrome/puppeteer/blob/v1.18.0/docs/api.md)
  * Chromium 76.0.3803.0 - [Puppeteer v1.17.0](https://github.com/GoogleChrome/puppeteer/blob/v1.17.0/docs/api.md)
  * Chromium 75.0.3765.0 - [Puppeteer v1.15.0](https://github.com/GoogleChrome/puppeteer/blob/v1.15.0/docs/api.md)
  * Chromium 74.0.3723.0 - [Puppeteer v1.13.0](https://github.com/GoogleChrome/puppeteer/blob/v1.13.0/docs/api.md)
  * Chromium 73.0.3679.0 - [Puppeteer v1.12.2](https://github.com/GoogleChrome/puppeteer/blob/v1.12.2/docs/api.md)
  * [All releases](https://github.com/GoogleChrome/puppeteer/releases)
```

And throw an error if no version is found

Reply:
I'll do this, adding few more changes to the file.